extra/benchmark/hashtables/hashtables.factor

```diff
@@ -7,7 +7,7 @@ QUALIFIED: assocs
 IN: benchmark.hashtables
 
 MEMO: strings ( -- str )
-    1 100 [a,b] 1 [ + ] accumulate nip [ number>string ] map ;
+    1 100 [a,b] 0 [ + ] accumulate* [ number>string ] map ;
```

Comment:
Oops, yes you're right. Thanks for catching it. If it doesn't change the
benchmark, maybe we can switch it to the more classical 1 3 6 10 [...]
sequence in another commit?

In extra/benchmark/hashtables/hashtables.factor
<https://github.com/factor/factor/pull/1548#discussion_r55370370>:

> @@ -7,7 +7,7 @@ QUALIFIED: assocs
>  IN: benchmark.hashtables
>
>  MEMO: strings ( -- str )
> -    1 100 [a,b] 1 [ + ] accumulate nip [ number>string ] map ;
> +    1 100 [a,b] 0 [ + ] accumulate* [ number>string ] map ;

This is however, the same thing:

IN: scratchpad 0 100 [a,b) 1 [ + ] cum-map 10 head .
{ 1 2 4 7 11 16 22 29 37 46 }

—
Reply to this email directly or view it on GitHub
<https://github.com/factor/factor/pull/1548/files#r55370370>.